codegen/src/main/java/software/amazon/awssdk/codegen/docs/PaginationDocs.java

```diff
@@ -238,6 +244,18 @@ private ClassName asyncPaginatedResponseType() {
         return poetExtensions.getResponseClassForPaginatedAsyncOperation(operationModel.getOperationName());
     }
 
+    private String getPaginatorLimitKeyName() {
+        return paginatorDefinition != null ? paginatorDefinition.getLimitKey() : "";
+    }
+
+    private CodeBlock noteAboutLimitConfigurationMethod() {
+        return CodeBlock.builder()
+                        .add("\n<p><b>Please notice that the configuration of $L won't limit the number of results "
+                             + "you get with the paginator, instead, it only limits the number of results in each page.</b></p>",
```

Comment:
Editorial Nit:
Please note that the configuration of $L won't limit the number of results you get with the paginator. It only limits the number of results in each page.